server/apparmor/apparmor_unsupported.go

```diff
@@ -3,6 +3,9 @@
 package apparmor
 
 const (
+	// DefaultApparmorProfile is the name of default apparmor profile name.
+	DefaultApparmorProfile = "ocid-default"
```

Comment:
this is repeated, let's have an apprmor file with no build tag to always be included (and don't duplicate consts)